src/gnosisSafe/config/production.js

```diff
@@ -0,0 +1,13 @@
+import {
```

Comment:
We don't have core DSA contracts on Rinkeby testnet. So I don't think this file is required.